kodi_addon_checker/check_dependencies.py

```diff
@@ -32,6 +39,15 @@ def check_addon_dependencies(report: Report, repo_addons: dict, parsed_xml, bran
             if required_addon not in ignore:
                 report.add(Record(
                     PROBLEM, "Required addon %s not available in current repository." % required_addon))
+            elif required_addon in VERSION_ATTRB:
+                try:
+                    version = VERSION_ATTRB[required_addon][branch_name]
+                    if LooseVersion(version) != LooseVersion(required_version):
+                        report.add(Record(WARNING, "For %s it is advised to set %s version to %s" %
+                                          (branch_name, required_addon, required_version)))
```

Comment:
This reports the wrong version which should be used.

Reply:
what do you mean by wrong version? like the version of the attribute ?

Reply:
e.g. `WARN: For helix it is advised to set xbmc.python version to 2.1.0`